src/EPR.RegulatorService.Frontend.Core/Enums/CountryName.cs

```diff
@@ -1,12 +1,18 @@
 namespace EPR.RegulatorService.Frontend.Core.Enums;
 
+using System.ComponentModel;
+
 public enum CountryName
 {
```

Comment:
This will be changed once we refactor the nationid/nationcode as part of other story